src/libs/actions/Report.js

```diff
@@ -486,6 +486,23 @@ function addAction(reportID, text, file) {
 
     // Convert the comment from MD into HTML because that's how it is stored in the database
     const parser = new ExpensiMark();
+
+    // Insert Email rule to ExpensMark rules
+    parser.rules.splice(2, 0, {
+        // For Markdown Email Link
+        name: 'MD-Email',
+        regex: /\[([\w\\s\\d!?&#;]+)\]\((([a-zA-Z0-9])+@[a-zA-Z]+?(\.[a-zA-Z]{2,6})+)\)/gim,
+        replacement: '<a href="mailto:$2">$1</a>'
+    });
+    parser.rules.join();
+    parser.rules.splice(3, 0, {
+        // For General Email Link
+        name: 'Email',
+        regex: /(?![^<]*>|[^<>]*<\\)([_*~]*?)(([a-zA-Z0-9])+@[a-zA-Z]+?(\.[a-zA-Z]{2,6})+)(?![^<]*(<\\pre>|<\\code>))/gim,
```

Comment:
Unfortunately, all lint warnings will appear on everyone's PRs. So we can't merge with any lint errors for the time being.